scripts/travis/install-requirements.sh

```diff
@@ -17,15 +22,21 @@ if [ "${TASK}" = 'compilepy3 ci-py3-unit' ] || [ "${TASK}" = 'ci-py3-integration
     # NOTE: We create the environment and install the dependencies first. This
     # means that the subsequent tox build / test command has a stable run time
     # since it doesn't depend on dependencies being installed.
-    if [ "${TASK}" = 'compilepy3 ci-py3-unit' ]; then
-        TOX_TASK="py36-unit"
+    build_tox_env() {
```

Comment:
Please revert the code logic as it was before without using this function.
It increases level of indirection, it's arguable if new version is easier comparing to previous one.

Additionally, with increased amount of diff it's harder to navigate what was the _actual_ change.

Reply:
Do the commit messages not help?

Reply:
I was confused several times by how the packs-tests were being run when the target task was clearly just for unit tests. Plus, because the packs-tests were hidden within the unit tests make target, the tox environment for the packs-tests was not actually getting built here as expected. So this change makes it clear that each of these tasks has a tox environment that needs to be built.

Reply:
@armab Has a good point here. Since it's a one-line function, this is a bit of indirection that isn't really useful.

Reply:
k. I was avoiding repeating the `tox -e ...` multiple times. Because a task can have multiple tox environments I needed a way to either loop over them, or just call it. I can un-DRY this here and repeat the tox calls I guess.

Reply:
Yeah, this both increases complexity (via indirection) and increases the number of lines in the file for very little benefit. Sorry if my comments elsewhere about the DRY-principle steered you this way, but in this case it's probably fine to "unindirect" the call to `tox`.

Reply:
removed the function and added comments explaining this.